library/src/skins/default/src/sass/themes/_light.scss

```diff
@@ -118,6 +118,14 @@
     // Used for active, focus, selected effects of --sakai-background-color-3
     --sakai-background-color-7: var(--sakai-color-gray--lighter-2);
 
+    // Default: A slightly darker shade of --sakai-background-color-7
+    // Used for active, focus, selected effects of --sakai-background-color-3
+    --sakai-background-color-8: var(--sakai-color-gray--lighter-1);
```

Comment:
Are we using `--sakai-background-color-8` ?

Reply:
Hi Kunal, we are not using ```--sakai-background-color-8```. However, I believe that using ```--sakai-background-color-9``` would be more appropriate for our needs than ```--sakai-background-color-8``` in this particular case. To ensure that there is no gap between ```--sakai-background-color-7``` and ```--sakai-background-color-9```, I have added ```--sakai-background-color-8``` to the file.

Reply:
Why not just use `--sakai-background-color-8` ? You can add the same variable to `8`?